<?xml version="1.0"?>

<launch>

  <arg name="sensor_id" default="lidar_1" />
  <arg name="sensor_number" default="2" />

  <node pkg="laser_detection_ros" type="laser_for_fusion" respawn="false" name="laser_collisions" output="screen">
    <param name="sensor_id" type="string" value="$(arg sensor_id)"/>
    <param name="sensor_number" type="int" value="$(arg sensor_number)"/>
  </node>
</launch>
